_source/user-guide/infrastructure-monitoring/explore-in-kibana.md

```diff
@@ -31,21 +29,19 @@ you can configure your own drilldown links.
 {:.no_toc}
 
 **Before you begin, you'll need**:
```

Comment:
That's not true technically. they don't HAVE to have variables, they can also hard-code the values to the link, although probably no one will ever do that.
So maybe change the phrasing to "we highly recommend on using variables" or something like that...